src/lib/multicall/hashData/hashDataAsync.ts

```diff
@@ -1,12 +1,11 @@
-import { uniqueId } from "lodash";
+import uniqueId from "lodash/uniqueId";
 
 import { sleep } from "lib/sleep";
 
-import HashDataWorker from "./hashData.worker";
 import { hashDataMap } from "./hashDataMap";
 import { promiseWithResolvers } from "lib/utils";
 
-export const hashDataWorker: Worker = new HashDataWorker();
+export const hashDataWorker: Worker = new Worker(new URL("./hashData.worker", import.meta.url), { type: "module" });
```

Comment:
this is dangerous change, need to find some library for initiating workers

Reply:
Discussed in IM: I performed some research, and it seems like Vite handles the new Worker() syntax in a specific way to ensure the reliable operation of workers. Also it's a recommended way to import workers according to vite documentation